### PR TITLE
Avoid repeated code to make the code more readable

### PR DESCRIPTION
**Pull request by valib**

### CODE REVIEW

File: src/main/java/net/pms/util/FileUtil.java [hunk 1]
```diff
@@ -537,57 +484,25 @@ public static String getFileNameWithRewriting(String f, File file) {
...
-			formattedName = formattedName.replaceAll(commonFileEndsCaseSensitive, "");
-			formattedName = formattedName.replaceAll("(?i)" + commonFileEnds, "");
-
-			// Capitalize the first letter of each word if the string contains no capital letters
-			if (formattedName.equals(formattedName.toLowerCase())) {
-				formattedName = convertLowerCaseStringToTitleCase(formattedName);
-			}
+			removeStuffAtTheEndOfFilename();
+			capitalizeFirstLetter();
 		} else if (formattedName.matches(".*\\((19|20)\\d\\d\\).*")) {
```
Comment (valib) [inline src/main/java/net/pms/util/FileUtil.java]: @SubJunk this repeated code is an intetion or bug?

**Comment by Nadahar** [comment]
Even though I haven't figured out what you've done yet (the GitHub diff is very confusing in some cases), I like the idea of making PRs like this :+1:

File: src/main/java/net/pms/util/FileUtil.java (hunk 1 above)
Comment (SubJunk) [inline src/main/java/net/pms/util/FileUtil.java]: @valib it looks unintentional, I've removed it now

**Comment by SubJunk** [comment]
I agree, I like this kind of PR, and this part of the code needs a tidy as it has gradually been growing. I'm looking through the changes now

**Comment by SubJunk** [comment]
I hope you don't mind but I'm doing some work in here too, if you do mind you can roll my changes back

**Comment by valib** [comment]
@SubJunk nice one. Originally this is your code so you now better than me what was the intention of the code and renaming made it more understandable.

**Comment by SubJunk** [comment]
:)

**Comment by SubJunk** [comment]
@UniversalMediaServer/developers does anyone have an issue with this being merged?

**Comment by Nadahar** [comment]
@SubJunk Not me, I don't really know or use the file name prettifying and have nothing useful to contribute to it.

**Comment by valib** [comment]
@SubJunk no objection from me.

**Pull request merged**